pkgs/development/interpreters/erlang/R19.nix

```diff
@@ -45,6 +45,8 @@ stdenv.mkDerivation rec {
 
   preConfigure = ''
     ./otp_build autoconf
+  '' + optionalString stdenv.isDarwin ''
+    substituteInPlace configure --replace "sw_vers" "/usr/bin/sw_vers"
```

Comment:
I just checked our stdenv sets `MACOSX_DEPLOYMENT_TARGET` to `10.10` so that's probably what we want to use.